packages/styled-components/components/image/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import sanitizeHtml from 'sanitize-html';
-import useImageSize from '@use-hooks/image-size';
+// import useImageSize from '@use-hooks/image-size';
```

Comment:
This is not isomorphically safe, so we'll probably have to figure out a different way of getting width.